src/styles/components/pictures-wrapper.scss

```diff
@@ -0,0 +1,80 @@
+.pictures {
+  margin: 40px 0 60px 120px;
+
+  &-wrapper {
+    margin: 0 240px 172px 120px;
+  }
+
+  &-about {
+    display: flex;
+    justify-content: space-between;
+
+    &__title {
+      font-family: $fontPlayfair;
+      font-size: $fontSizeXL;
+      mix-blend-mode: normal;
+      font-weight: 700;
```

Comment:
Can you use the shorthand property for the font?

Reply:
Impossible due to selectors